File: week15-1.py
class Solution:
    def duplicateNumbersXOR(self, nums: List[int]) -> int:
        table={}
        for num in nums:
            if num in table:
                table[num]+=1
            else:
                table[num]=1
        #print(table)
        ans=0
        for num in table:
            if table[num]==2:
                ans-ans^num
        return ans

File: week15-1.c
//week15-1
int duplicateNumbersXOR(int* nums, int numsSize) {
    int table[51] ={};
    for(int i=0; i<numsSize; i++ ){
        table[nums[i]]++;
    }
    int ans = 0;
    for(int i=1; i<50; i++){
        if(table[i]==2) ans = ans ^i;
    }
    return ans;
}
